tests/test.mk

```diff
@@ -58,6 +58,14 @@ themispp_test: $(THEMISPP_TEST_OBJ)
 	@echo -n "link "
 	@$(BUILD_CMD)
 
+rustthemis_integration_tools:
```

Comment:
could you please check that `make clean` cleans up rust files as well?

Reply:
I've extended the `make clean` target to remove Rust's `target` directory and all those `tools/rust/*.rust` files.

It's not the cleanest solution, but this should be okay for a general use case. (Alternatively, it is possible to set CARGO_TARGET_DIR and have Cargo put all its Rust stuff into the same build directory as the rest of the build system uses. But we will need to make sure that this environment variables is set everywhere.)

Reply:
i think it's good for now, we might think about adding `CARGO_TARGET_DIR` during Makefile refactoring